src/engine/strat_engine/setup.rs

```diff
@@ -206,6 +208,20 @@ pub fn get_blockdevs(pool_save: &PoolSave, devnodes: &[PathBuf]) -> EngineResult
         let actual_size = try!(blkdev_size(&try!(OpenOptions::new().read(true).open(dev))))
             .sectors();
 
+        match actual_size.cmp(&bda.dev_size()) {
```

Comment:
We do not have any code that uses it yet. The way we should use it is run some sort of check on the pool after it has been reconstructed.

My point is that we do not have to act like frantic physicists who are trying to make certain that we actually just, however briefly, obtained the 125th element. Each blockdev necessarily knows its actual capacity and its recorded size. We can use this information at any point, to take whatever action we desire. This information is not transient, it is not important that it was first obtained at this point in the setup code, it is a constant known fact about every blockdev. We should trreat it that way.